### PR TITLE
Add multiple nests idealized 2TC test case

### PR DESCRIPTION
**Pull request by JosephMouallem**
Adds a new runscript to test the multiple same level an telescoping nest framework in the solo core. The test case consists of two idealized TCs running at a C48 global resolution with three nests:
1- One nest over the first TC
2- Two nests (one telescoping) over the second TC

### CODE REVIEW

**Comment by bensonr** [comment]
@JosephMouallem  - the scripts are being updated to handle both ascii and yaml inputs that FMS now accepts.  You can find the necessary updates in PR #34.

File: RTS/CI/C48.solo.2TC.3nest [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+     !gfs_phil = $gfs_phil
+     !nggps_ic = $nggps_ic
+     !mountain = $mountain
+     ncep_ic = .F.
+     d_con = $d_con
+     hord_mt = 6
+     hord_vt = 6
+     hord_tm = 6
+     hord_dp = 6
+     hord_tr = 8
```
Comment (lharris4) [inline RTS/CI/C48.solo.2TC.3nest:356]: For tropical cyclones I would recommend using hord=5 and hord_tr = -5.

File: RTS/CI/C48.solo.2TC.3nest [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+#* If not, see <http://www.gnu.org/licenses/>.
+#***********************************************************************
+#
+#  DISCLAIMER: This script is provided as-is for the purpose of continuous
+#              integration software regression testing and as such is
+#              unsupported.
+#
+#SBATCH --ntasks=96
+
+#Dry baroclinic wave
```
Comment (lharris4) [inline RTS/CI/C48.solo.2TC.3nest:29]: May wish to delete or change this line.

File: RTS/CI/C48.solo.2TC.3nest [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+#Also demonstrates the plev diagnostic capability
```
Comment (lharris4) [inline RTS/CI/C48.solo.2TC.3nest:30]: This one too

File: RTS/CI/C48.solo.2TC.3nest [hunk 4]
```diff
@@ -0,0 +1,540 @@
...
+/
+
+EOF
+
+
+
+# Nests namelists
+##################
+
+for ((counter=0; counter<$num_nest; counter++)); do
```
Comment (lharris4) [inline RTS/CI/C48.solo.2TC.3nest:413]: This iteration over nests is a very nice addition that simplifies the runscript

**Comment by JosephMouallem** [comment]
Thanks Rusty and Lucas. I made the suggested changes and tested a run with yaml as well by compiling FMS through ./COMPILE yaml and setting use_yaml=".T." following the recommendation of #34. All works fine on C5.

**Review by lharris4 — APPROVED** [review]
Thank you. This should be about ready to merge in.

File: RTS/CI/C48.solo.2TC.3nest [hunk 5]
```diff
@@ -0,0 +1,616 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (bensonr) [inline RTS/CI/C48.solo.2TC.3nest:320]: In the future we might want to remove all of these empty lines.

**Review by bensonr — APPROVED** [review]

**Comment by laurenchilutti** [comment]
I'll merge this in now and work on adding it to the automated CI.

**Pull request merged**